README.md

```diff
@@ -1,16 +1,767 @@
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Explain what constitutes an expression in JavaScript
+* Organize code using block statements
+* Describe the difference between truthy and falsy values
+* Learn to use conditional statements
 
 ## Introduction
 
-## SWBAT 1
+Consider this real-life scenario: If you're hungry, you make a meal. If the
+traffic light is green, you press the car's gas pedal. If a bill is due, then
+you pay the bill. This breaks down into a lot of conditional choices:
 
-## SWBAT 2
+- `if` hungry :arrow_right: make a meal.
+  - `else` :arrow_right: don't make a meal.
+- `if` light is green :arrow_right: press gas pedal.
+  - `else` :arrow_right: press brake pedal.
+- `if` it's the first of the month :arrow_right: pay the bill.
+  - `else` :arrow_right: don't pay the bill.
+
+Writing code involves the same type of logic -- we only want an action to happen
+_if_ a certain condition is met. In the programming world, this is called
+**control flow** because, it helps _control_ the _flow_ of an application.
+
+Let's go over a few concepts that provide the syntactic underpinnings.
+
+## Explain What Constitutes an Expression in JavaScript
+
+A JavaScript expression is **a unit of code that returns a value**. Primitive
+values are expressions because they resolve to a value:
+
+```js
+9;
+// => 9
+
+('Hello, world!');
+// => "Hello, world!"
+
+false;
+// => false
+```
+
+So are arithmetic and string operations. This code resolves to the number `64`:
+
+```js
+8 * 8;
+// => 64
+```
+
+This resolves to the string `"Hello, world!"`:
+
+```js
+'Hello, ' + 'world!';
+// => "Hello, world!"
+```
+
+Same for comparison and assignment operations. This comparison resolves to the
+boolean `true`:
+
+```js
+2 > 1;
+// => true
+```
+
+Variable declarations are **NOT** expressions...
+
+```js
+let answer;
+```
+
+Variable assignments **ARE**, resolving to the assigned value (`42`, in this
+case):
+
+```js
+answer = 42;
+// => 42
+```
+
+Finally, references to variables are also expressions, resolving to the value
+contained in the variable:
+
+```js
+const fullName = 'Ada Lovelace';
+
+fullName;
+// => "Ada Lovelace"
+```
+
+## Organize Code Using Block Statements
+
+A _block statement_ is a pair of curly braces (`{ }`) used to group JavaScript
+statements. It plays a role in conditional statements, loops, and
+functions.
+
+```js
+{
+	('This line is a JavaScript statement nested inside a block statement!');
+
+	// This is also a statement nested inside a block:
+	5 * 5 - 5;
+
+	// And so are these:
+	const weCan = 'group multiple statements';
+
+	const suchAs = 'these variable declarations';
+
+	const insideA = 'block statement.';
+}
+// => 20
+```
+
+Block statements return the value of the last evaluated expression inside the
+curly braces. Remember, the variable declarations are _not_ expressions, so the
+value of `5 * 5 - 5` is returned.
+
+**Note**: The statement above _implicitly_ returns 20 (the value returned by
```

Comment:
I've always felt this note is kind of more confusing than not. I'm not sure how to make it less so. Also, we're covering arrow functions earlier in this particular course. We probably need to remove language about "covering things later" (or earlier) so lessons will be a bit more modular.